scripts/us_epa/air_emissions_inventory/common/us_air_pollution_emission_trends.py

```diff
@@ -0,0 +1,183 @@
+
+    def aggregate_columns(self, df: pd.DataFrame, filter_values: list,
+                          measurement_method: str) -> pd.DataFrame:
+        """
+        Aggregates the columns based on SV
+
+        Args: 
+            df (pd.DataFrame): df as the input, to aggregate values
+
+        Returns:
+            df (pd.DataFrame): modified df as output
+        """
+        for i in filter_values:
+            df = df.drop(df[(df['SV_TEMP'].str.contains(i))].index)
+
+        # Replacing the columns for grouping as per
+        # StationaryFuelCombustion -    FuelCombustionElectricUtility
+        #                               FuelCombustionIndustrial
+        #                               EPA_FuelCombustionOther
+        # IndustrialAndOtherProcesses - ChemicalAndAlliedProductManufacturing
+        #                               MetalsProcessing
+        #                               PetroleumAndRelatedIndustries
+        #                               EPA_OtherIndustrialProcesses
+        #                               SolventUtilization
+        #                               StorageAndTransport
+        #                               WasteDisposalAndRecycling
+        # Transportation -              OnRoadVehicles
+        #                               NonRoadEnginesAndVehicles
+
+        df.loc[:, ('SV_TEMP')] = df['SV_TEMP'].replace(_SOURCE_GROUPS,
+                                                       regex=True)
+        df = df.groupby(['year', 'geo_Id', 'SV_TEMP']).sum().reset_index()
+        df['Measurement_Method'] = measurement_method
+        return df
+
+    def data_standardize(self, df: pd.DataFrame, column_name: str,
+                         standardize_values: dict) -> pd.DataFrame:
+        """
+        Replaces values of a single column into true values
+        from metadata.
+
+        Args:
+            df (pd.DataFrame): df as the input, to change column values
+
+        Returns:
+            df (pd.DataFrame): modified df as output
+        """
+        df = df.replace({column_name: standardize_values})
+        return df
+
+    def _parse_source_files(self):
+        pass
+
+    def generate_tmcf(self) -> None:
+        """
+        This method generates TMCF file w.r.t
+        dataframe headers and defined TMCF template.
+
+        Args:
+            None
+
+        Returns:
+            None
+        """
+        # Writing Genereated TMCF to local path.
+        with open(self._tmcf_file_path, 'w+', encoding='utf-8') as f_out:
+            f_out.write(self._tmcf_template.rstrip('\n'))
+
+    def generate_mcf(self, df: pd.DataFrame = None):
+        """
+        This method generates MCF file w.r.t
+        dataframe headers and defined MCF template
+
+        Args:
+            df_cols (list) : List of DataFrame Columns
+
+        Returns:
+            sv_replacement (dict) : Dictionary to replace SV names 
+                                    with SV generator names
```

Comment:
This function doesn't return anything. Does it need to?

Reply:
Hi Ajai, this function was initially made by me where this was needed. However, later the whole code was updated and the function was changed. Now it does not need to return anything. I will update the docstring.